CuriousKatie/CuriousKatie/Extensions.swift

```diff
@@ -16,3 +16,11 @@ extension Array where Element: Hashable {
         return Array(thisSet.symmetricDifference(otherSet))
     }
 }
+
+
+extension Interest {
```

Comment:
If this is the Equatable protocol conformance implementation, you should also move that declaration here...
Ex:
```suggestion
extension Interest: Equatable {
```

There's also no real reason this extension shouldn't be in Interests.swift
(Creating a separate file creates a dependency between this **Extensions.swift** and **Interest.swift**; JUST for the Interest structure to work.  In **this** project, it's just a code-smell.  In another project, it WILL be the reason you have to do a bunch of weird stuff like copy & paste whole files of duplicated code just to access this 1 extension's functionality for 1 object.)